### PR TITLE
Yuyu3/change amlk8s to amlarc

### PR DESCRIPTION
**Pull request by yuyue9284**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your PR is merged into master branch, a new PR will be created to update `src/index.json` automatically.  
The precondition is to put your code inside this repo and upgrade the version in the PR but do not modify `src/index.json`.

### CODE REVIEW

File: src/k8s-extension/HISTORY_private.rst [hunk 1]
```diff
@@ -3,6 +3,11 @@
 Release History
 ===============
 
+0.4.0-beta.2
```
Comment (jonathan-innis) [inline src/k8s-extension/HISTORY_private.rst:6]: Are these meant to be private changes or changes for public-preview?

**Review by jonathan-innis — CHANGES_REQUESTED** [review]

File: src/k8s-extension/HISTORY_private.rst (hunk 1 above)
Comment (yuyue9284) [reply src/k8s-extension/HISTORY_private.rst:6]: This change is meant to be for public preview, in that case should I just open a PR against the public branch or open PRs for both public and private branch?

Comment (yuyue9284) [reply src/k8s-extension/HISTORY_private.rst:6]: Also created a PR against the public branch. Thanks.
https://github.com/AzureArcForKubernetes/azure-cli-extensions/pull/42

Comment (jonathan-innis) [reply src/k8s-extension/HISTORY_private.rst:6]: Just public, we handle downstream merging so when you are making a change that should be public, please make it against k8s-extension/public

**Comment by jonathan-innis** [comment]
Closing this as it is a duplicate

**Pull request closed without merging**